### ISSUE TITLE
Login function with Password-Hash function Error

### ISSUE THREAD

**Issue by JacobYim** [JacobYim/CSE442-BuyNSell#56]
In the branch `password-hash`,

When run the `singin_acceptance_test.py`, some test has been failed

```
Junghwan-ui-MacBook-Pro:test junghwanyim$ python3 signin_acceptance_test.py 
Pass test1
Pass test2
Pass test3
Pass test4
Fail test5 : string having ";" case is not blocked.
Fail test6 : string having "." case is not blocked.
Pass test7
Fail test8 : valid signin is failed
Fail test9 : valid signin is failed
Pass test10
```

**Comment by JacobYim** [comment]
It is because of the position of hashing password.

Change app.post part to

app.post('/signup.html', (req,res) => {
  var userId = String(req.body['name']);
  var email = String(req.body['email']);
  var password = String(req.body['password']);
  var ubid = String(req.body['ubid']);

  console.log("Typed :",userId, email, password,ubid);
  if ((userId != '' && userId != ' ' && !userId.includes(';') && !userId.includes('.')&& !userId.includes('=')) &&
      (email != '' && email != ' ' && !email.includes(';') && !email.includes('=') && email.includes('@') && email.includes('.')) && 
      (password != '' && password != ' ' && !password.includes(';') && !password.includes('.') && !password.includes('=')) &&
      (ubid != '' && ubid != ' ' && ubid.length == 8 && !ubid.includes(';') && !ubid.includes('='))){
        
        //secures password here
        password = passowrdHasher(password);
        console.log("Password is Secure.......................")
        
        db.query('insert into user_profile(ubid, email, username, password) values(\''+ubid+'\',\''+email+'\',\''+userId+'\',\''+password+'\')', function (err, rows, fields) {
      if (!err) {
          console.log(rows)
          res.send('success');
          } else {
          res.send('err : ' + err);
      }
    });
  }else{
    res.send('wrong approach');
  }
});

**Comment by JacobYim** [comment]
Even though changed like that, the acceptance test is not passed.

```
Pass test1
Pass test2
Pass test3
Pass test4
Pass test5
Pass test6
Pass test7
Fail test8 : valid signin is failed
Fail test9 : valid signin is failed
Pass test10
```
it will be discussed at #57

**Comment by JacobYim** [comment]
It has been solved at #58
